src/resources.rs

```diff
@@ -58,6 +58,36 @@ enum Repr {
   TcpStream(tokio::net::TcpStream),
 }
 
+pub fn get_resource_table_entries() -> Vec<(i32, String)> {
+  let table = RESOURCE_TABLE.lock().unwrap();
+
+  let tuples = table
+    .iter()
+    .map(|(key, value)| (key.clone(), inspect_repr(&value)))
+    .collect();
+
+  tuples
+}
+
+#[test]
+fn test_get_resource_table_entries() {
+  assert_eq!(get_resource_table_entries().len(), 3);
+  // TODO: add asserts for add_fs_file, add_tcp_listener, add_tcp_stream
```

Comment:
Maybe check some of the entries? 
```
let entries = get_resource_table_entries();
assert_eq!(entries[0], (0, "stdin")
```

Reply:
That's a good idea, but HashMap doesn't guarantee that first entry will be the first one added, does it?

Using this:
```
#[test]
fn test_get_resource_table_entries() {
  let entries = get_resource_table_entries();
  assert_eq!(entries.len(), 3);
  assert_eq!(entries[0], (0, String::from("stdin")));
  assert_eq!(entries[1], (1, String::from("stdout")));
  assert_eq!(entries[2], (2, String::from("stderr")));
}
```
I get this:
```
---- resources::test_get_resource_table_entries stdout ----
thread 'resources::test_get_resource_table_entries' panicked at 'assertion failed: `(left == right)`
  left: `(2, "stderr")`,
 right: `(0, "stdin")`', ../../src/resources.rs:76:3
```

Reply:
Call `sort()` inside of `get_resource_table_entries()`